storage_controller/src/service.rs

```diff
@@ -6353,8 +6359,14 @@ impl Service {
         &self,
         shard: &mut TenantShard,
         nodes: &Arc<HashMap<NodeId, Node>>,
+        high_priority: bool,
```

Comment:
an enum would be better here, `Priority::Normal`, `Priority::High` or something. but it's okay for a hotfix.